qa/L1_pytorch_mcore_integration/test.sh

```diff
@@ -40,17 +54,17 @@ ${MCORE_PATH}/pretrain_gpt.py
 --hidden-size 128
 --num-attention-heads 8
 --seq-length 128
---max-position-embeddings 2048
+--max-position-embeddings 128
 --micro-batch-size 1
 --global-batch-size 8
 --train-iters 10
 --eval-iters 10
 --lr 1e-4
 --mock-data
---vocab-file /data/gpt3/pile-cc1-cc2-shuf/bpe/gpt2-vocab.json
---merge-file /data/gpt3/pile-cc1-cc2-shuf/bpe/gpt2-merges.txt
+--vocab-file ${VOCAB_FILE}
+--merge-file ${TE_PATH}/qa/L1_pytorch_mcore_integration/merges.txt
```

Comment:
Not sure what's the purpose of this file, but is it okay for it to only have version info?

Reply:
I'm treating `merges.txt` as a magic config for the tokenizer: https://github.com/huggingface/transformers/issues/1083#issuecomment-524303077